client/app/utils/transitImmutableConverter.js

```diff
@@ -1,6 +1,33 @@
 import transit from 'transit-js';
 import Immutable from 'immutable';
 
+export const Image = Immutable.Record({
+  type: ':square',
+  height: 408,
+  width: 408,
+  url: null,
+});
+
+const ImageRefs = Immutable.Record({
+  square: new Image(),
+  square2x: new Image(),
+});
+
+const toImage = (rep) => {
+  const knownStyles = {
+    ':square': 'square',
+    ':square_2x': 'square2x',
+  };
+  const data = rep[1];
```

Comment:
I had to remove this in my branch, but it should probably be done here instead:
https://github.com/sharetribe/sharetribe/pull/2582/files?diff=unified#diff-17b1cd2a70336c85378e752ce85a7bd9L16